runtime/resolvers/testdata/metrics_sql_postgres_to_clickhouse.yaml

```diff
@@ -0,0 +1,57 @@
+connectors:
```

Comment:
1. Can you merge this file with `metrics_sql_postgres_to_duckdb.yaml`? We support using multiple OLAPs in one test file (see `metrics_comparisons.yaml` for an example). Breaking into multiple files is okay when the test files grow very long, but right now it will be pretty short even if combined.
2. The test files are called `metrics_sql_postgres_to_<olap>.yaml`, but doesn't have any tests related to our metrics SQL. Consider calling it just `postgres_connector.yaml`.